libraries/standard/json/cbmc/proofs/JSON_Search/glue.h

```diff
@@ -0,0 +1,27 @@
+#ifndef GLUE_H_
+#define GLUE_H_
+
+#include "json.h"
+#include "skipGeneric.h"
+
+bool_ skipAnyScalar( const char * buf,
```

Comment:
Can we please document these?

Reply:
They are replacements for the functions of the same name from json.c, so the documentation is in json.c.  I'll add a comment to that effect.

Reply:
added comment about doc location